ui/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
I am new to node -- what's the difference between `package.json` and `package-lock.json`? Should both be version controlled in git?

Reply:
Also new to node - looking at the docs, package-lock is auto-generated on build (after deps are added) and it basically ensures consistency during building by new developers, otherwise secondary dependencies may change version and cause unexpected and difficult to debug breakages.

Accordingly, it should be checked in (but all edits are managed automatically by npm).